HealthSDK/GiniHealthSDKExample/GiniHealthSDKExample/DebugMenuViewController.swift

```diff
@@ -58,6 +59,17 @@ class DebugMenuViewController: UIViewController {
     private var closeButtonSwitch: UISwitch!
     private lazy var closeButtonRow: UIStackView = stackView(axis: .horizontal, subviews: [closeButtonOptionLabel, closeButtonSwitch])
 
+    private lazy var qrFilenameOptionLabel: UILabel = rowTitle("QR PDF Filename")
+    private lazy var qrFilenameTextField: UITextField = {
```

Comment:
As well here shareWithFilenameTextField and make it more generic if we decide to remove QRCode?